docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md

```diff
@@ -32,9 +40,9 @@ wget https://nebula-graph.oss-accelerate.aliyuncs.com/jar-packages/sst.generator
 This section includes the following steps:
 
 1. Create a graph space and its schema in Nebula Graph
-1. Write data files
-1. Write input source mapping file
-1. Import data
+2. Write data files
```

Comment:
why?

Reply:
I don't really get your point but this is an ordered list, attached with the preview:
![image](https://user-images.githubusercontent.com/42762957/70197154-df043a80-1745-11ea-9235-b3a08c8fb806.png)